imaging/rc.local.copyme

```diff
@@ -1,14 +1,6 @@
 #!/bin/bash
-# Copy 1 of the following contents to the end of rc.local:
+# Copy the following contents to the end of rc.local:
 
-# OpenCV Python
-touch /home/warg/record.log  # To confirm script has run
-cd /home/warg/model-training/
-source venv/bin/activate
-cd imaging/
-python -m imaging
+/home/warg-rpi4b/model-training/imaging/run.sh
```

Comment:
`/home/warg/...`

Reply:
Just curious, is there a specific reason we want to use `warg`? Is it for generality?